invictus_dashboard/src/app/dashboard/components/dashboard/dashboard.component.html

```diff
@@ -20,18 +20,19 @@ <h4>{{ startString }} - {{ endString }} Sprint</h4>
             class="progress-bar bg-primary"
             role="progressbar"
             [style.width.%]="sprintProgress"
+            style="overflow: hidden"
             aria-valuenow="0"
             aria-valuemin="0"
             aria-valuemax="100"
           >
-            {{ sprintProgress }}% complete
+            {{ sprintProgress }}%
           </div>
         </div>
       </div>
     </div>
     <div class="row rounded-bottom container-fluid">
       <p class="text-warning">{{ errMsg }}</p>
-      <ul *ngIf="false">
+      <ul *ngIf="true">
```

Comment:
You can put this in a readme. that is probably a better spot for them.

Reply:
I'll shift this somewhere else, thanks.